atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/CheckAndSetCompatibility.java

```diff
@@ -31,6 +31,11 @@ public interface CheckAndSetCompatibility {
      */
     boolean supportsCheckAndSetOperations();
 
+    @Value.Default
+    default boolean supportsMultiCheckAndSetOperations() {
+        return false;
+    }
+
```

Comment:
🏅 uff we need to wire this into TransactionServices code

Reply:
yea